mava/wrappers/smac.py

```diff
@@ -64,10 +65,11 @@ def reset(self) -> dm_env.TimeStep:
             dm_env.TimeStep: dm timestep.
         """
         # Reset the environment
-        self._environment.reset()
+        if not self._is_resetted:
```

Comment:
Also would be nice to be explicit in the docstring, "Resets the env, if it was not reset on the previous step"

Reply:
Agreed and fixed.